### ISSUE TITLE
IE11 object missing most fields

### ISSUE THREAD

**Issue by dmt0** [lifaon74/url-polyfill#65]
Here's my example URL: 
`https://dash.local/Auth/o/authorize/?response_type=token&client_id=eC5Tuezee6eebahn5xo9moo4edai2weepheethah&redirect_uri=https://dash.local/Manager/oauth2-redirect&state=Manager`

When I parse it in Chrome:
```
hash: ""
host: "dash.local"
hostname: "dash.local"
href: "https://dash.local/Auth/o/authorize/?response_type=token&client_id=eC5Tuezee6eebahn5xo9moo4edai2weepheethah&redirect_uri=https://dash.local/Manager/oauth2-redirect&state=Manager"
origin: "https://dash.local"
password: ""
pathname: "/Auth/o/authorize/"
port: ""
protocol: "https:"
search: "?response_type=token&client_id=eC5Tuezee6eebahn5xo9moo4edai2weepheethah&redirect_uri=https://dash.local/Manager/oauth2-redirect&state=Manager"
searchParams: URLSearchParams {}
username: ""
__proto__: URL
```

When I parse it in IE11 (that thing won't let me copy/paste...):

![image](https://user-images.githubusercontent.com/2717746/102546861-49082d00-4086-11eb-938f-e2399b338c95.png)

host, hostname, protocol are all missing.

**Comment by dmt0** [comment]
Got things working with this one for now: https://github.com/Financial-Times/polyfill-library/blob/master/polyfills/URL/polyfill.js

**Comment by sashaweiss** [comment]
FWIW, was just experimenting with this polyfill and tried the URL you provided above, and it seems to parse correctly:

<img width="1396" alt="image" src="https://user-images.githubusercontent.com/9013117/108435881-3d102700-71ff-11eb-8c1e-12c3fbf089b3.png">

**Comment by dmt0** [comment]
Yeah, that's weird. Gonna give it another shot when I have my hands on IE again.

**Comment by jimmywarting** [comment]
ie = dead
